docs/envvars.rst

```diff
@@ -63,6 +63,11 @@ applicable.
         writing ``$COMPLETIONS_DISPLAY = None`` and ``$COMPLETIONS_DISPLAY = 'none'`` is equivalent.
         
         (Only usable with SHELL_TYPE=prompt_toolkit)
+    * - COMPLETIONS_MENU_ROWS
+      - ``3``
```

Comment:
I might advocate a bigger number than 3, but maybe that's just me.  Maybe we could use 5, as a happy medium between was prompt_toolkit does (7, I think) and this value?